vertx-web/src/main/java/io/vertx/ext/web/handler/impl/UserHolder.java

```diff
@@ -64,8 +64,6 @@ public void writeToBuffer(Buffer buffer) {
 
     synchronized (this) {
       user = context != null ? context.user() : this.user;
-      // clear the context as this holder is not in a request anymore
-      context = null;
```

Comment:
I didn't find the reason why context = null here, and maybe there is any potential risk.